src/utils/apis/LoginAPI.js

```diff
@@ -2,6 +2,12 @@ import {recruitmentServiceBasicAPILink} from "./APILinks";
 
 export const loginAPI = {
 
+    authTokenKey: "x-auth-token",
+    refreshTokenKey: "x-refresh-token",
```

Comment:
Napisz jakieś testy integracyjne z backendem do tych headerów - jestem na 99% pewny, że to walnie undefined.